apps/scan/backend/test/helpers/custom_helpers.ts

```diff
@@ -228,29 +228,18 @@ export async function scanBallot(
       initialBallotsCounted === 0 || (await apiClient.getConfig()).isTestMode,
   });
 
-  const interpretation: SheetInterpretation = {
-    type: 'ValidSheet',
-  };
-
   mockScanner.scan.mockResolvedValueOnce(ok(await ballotImages.completeBmd()));
   await apiClient.scanBallot();
   mockScanner.getStatus.mockResolvedValue(ok(mocks.MOCK_READY_TO_EJECT));
   await waitForStatus(apiClient, {
     state: 'ready_to_accept',
-    interpretation,
     ballotsCounted: initialBallotsCounted,
     canUnconfigure: true,
+    interpretation: { type: 'ValidSheet' },
   });
+
   await apiClient.acceptBallot();
   mockScanner.getStatus.mockResolvedValue(ok(mocks.MOCK_NO_PAPER));
-  await waitForStatus(apiClient, {
-    ballotsCounted: initialBallotsCounted + 1,
-    state: 'accepted',
-    interpretation,
-    canUnconfigure: true,
-  });
```

Comment:
This is an existing (pre-continuous-export) source of test flakiness. The scanner state machine sometimes transitions so quickly from the `accepted` state to the `no_paper` state in tests that we never detect the `accepted` state. Checking for the `accepted` state is unimportant in the context of this `scanBallot` helper. We can still be sure that the ballot was accepted because of the `ballotsCounted: initialBallotsCounted + 1` in the `no_paper` state.

Reply:
Does this affect the user experience? If we transition too quickly from the `accepted` state is there a screen that doesn't get rendered?

Reply:
No effect here on actual UX since this is just a test helper. In a number of tests, we intentionally reconfigure state machine delays to trigger certain states and/or to speed tests up, making it possible to quickly pass the accepted state (in a way that we wouldn't outside of tests)

Reply:
Missed the filepath, thanks!